kitchen.yml

```diff
@@ -6,7 +6,7 @@ provisioner:
   # the ohai_plugin from the dovecot cookbook fails idempotency checks
   # enforce_idempotency: true
   # multiple_converge: 2
-  deprecations_as_errors: true
+  deprecations_as_errors: false
```

Comment:
You should add a comment here as to which cookbook is causing us to change this.